src/python/ddapp/valvedemo.py

```diff
@@ -1195,7 +1196,7 @@ def addManipulation(func, name, parent=None):
         addTask(rt.PlanPostureGoal(name='plan arm up pregrasp', postureGroup='General',
                                    postureName='arm up pregrasp', side=side), parent=preturn)
         addTask(rt.CommitManipulationPlan(name='execute manip plan',
-                                          planName='arm up pregrasp plan'), parent=preturn)
+                                          planName='arm up pregrasp posture plan'), parent=preturn)
```

Comment:
This is the real change in this PR - the rest is autoformatting. Key to note that PlanPostureGoal plans are now '...posture plan' instead of '... plan'. The other issues we are seeing with valveDemo are SCS based (doesnt adhere to joint limits, does not correctly execute trajectories)